src/test/prediction_service_test.cpp

```diff
@@ -1572,4 +1572,21 @@ TYPED_TEST(TestPredict, InferenceWithNegativeShapeDynamicParameter) {
     ASSERT_NE(modelInstance->infer(&request, &response, modelInstanceUnloadGuard), ovms::StatusCode::OK);
 }
 
+TYPED_TEST(TestPredict, InferenceWithStringInputs_positive) {
+    if (typeid(typename TypeParam::first_type) == typeid(ovms::InferenceRequest))
+        GTEST_SKIP() << "String inputs not supported for C-API";
+    typename TypeParam::first_type request;
+    std::vector<std::string> inputStrings = {"String_123", "String"};
+    prepareInferStringRequest(request, PASSTHROUGH_MODEL_INPUT_NAME, inputStrings);
+    ovms::ModelConfig config = PASSTHROUGH_MODEL_CONFIG;
+    config.setBatchingParams("auto");
```

Comment:
We can perform the same test for shape `"(-1, -1)"`, static `"(2, 11)"` and range `"(1:3,9:15)"`.
We can perform negative test for static `"(3,12)"` and range `"(3:6,20:30)"`